G/GDAL/build_tarballs.jl

```diff
@@ -6,8 +6,8 @@ name = "GDAL"
 upstream_version = v"3.5.1"
 version_offset = v"0.0.1"
 version = VersionNumber(upstream_version.major * 100 + version_offset.major,
-                        upstream_version.minor * 100 + version_offset.minor,
-                        upstream_version.patch * 100 + version_offset.patch)
+    upstream_version.minor * 100 + version_offset.minor,
+    upstream_version.patch * 100 + version_offset.patch)
```

Comment:
This seems to be a whitespace-only change which has the only effect of making git-blaming harder?